src/content/docs/auto-telemetry-pixie/get-started-auto-telemetry-pixie.mdx

```diff
@@ -68,13 +68,53 @@ If you arrived in the guided installation process by following a link from Pixie
 
 2. Click **Continue**.
 3. Select the data you want to gather, observe, and debug, and click **Continue**.
-4. On the **Choose install method** page, copy the Helm command that's provided, and then run it on your command line. See [this page about installing the Kubernetes integration using Helm](/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#h2-compatibility-and-requirements) to learn more about the process.
-
-  Helm installs a bundle containing the New Relic infrastructure agent, an integration to gather Prometheus metrics and Kubernetes events, and the Pixie integration. The deployment takes a few minutes to complete. 
-
-    ![Pixie-Helm](./images/helm-pixie.png "Pixie Helm")
-
-    To see the status of the install to the cluster, run `kubectl get pods -n newrelic`. 
+4. On the **Choose install method** page, select either **Helm** or **manifest**. 
+  
+  <CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Helm method"
+  >
+  If you choose Helm, copy the Helm command that's provided, and then run it on your command line. See [this page about installing the Kubernetes integration using Helm](/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#h2-compatibility-and-requirements) to learn more about the process.
+
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/pixie-labs/pixie/main/k8s/vizier_deps/base/nats/nats_crd.yaml && \
+helm repo add newrelic https://helm-charts.newrelic.com && \
+kubectl create namespace newrelic ; helm upgrade --install newrelic-bundle newrelic/nri-bundle \
+ --set global.licenseKey=2c4e0ff30d8f9a8aaf71e47f419917a4b944NRAL \
+ --set global.cluster=tester \
+ --namespace=newrelic \
+ --set newrelic-infrastructure.privileged=true \
+ --set ksm.enabled=true \
+ --set prometheus.enabled=true \
+ --set kubeEvents.enabled=true \
+ --set logging.enabled=true \
+ --set newrelic-pixie.enabled=true \
+ --set newrelic-pixie.apiKey=bcbd6fc2-930d-4840-a5c9-1299a73d578d \
+ --set pixie-chart.enabled=true \
+ --set pixie-chart.deployKey=a20679e7-2bc5-4bb5-b992-d82aa8c5b5c8 \
+ --set pixie-chart.clusterName=tester 
+ ```
+
+  Helm installs a bundle containing the New Relic infrastructure agent, an integration to gather Prometheus metrics and Kubernetes events, and the Pixie integration. The deployment takes a few minutes to complete. To see the status of the install to the cluster, run `kubectl get pods -n newrelic`. 
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="manifest method"
+  >
+  If you choose manifest, run the following command in your console, and insert the path to where you downloaded the manifest. If you're running your Kubernetes cluster in the cloud, see the additional steps in the [Kubernetes docs](/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure/#cloud-platforms). 
```

Comment:
"insert the path to where" --> "insert the path where" 
The "to" feels unnecessary. A little thing.